sources/packages/backend/apps/test-db-seeding/src/seed-executors/data-seed.decorator.ts

```diff
@@ -0,0 +1,56 @@
+import { SetMetadata } from "@nestjs/common";
+
+// metaDataKey for the provider class.
+export const DATA_SEED = Symbol.for("DATA_SEED");
+// metaDataKey for the provider method.
+export const DATA_SEED_METHOD = Symbol.for("DATA_SEED_METHOD");
+
+// Seed priority order.
+export enum SeedPriorityOrder {
+  Priority1 = 1,
+  Unknown = Number.MAX_SAFE_INTEGER,
+}
+
+// Provided interface.
```

Comment:
Please change to method comments. Is this way VSCode will not show the comments.
Same for line 20.